app/uk/gov/hmrc/crdlcacheadminfrontend/codeLists/models/CodeListEntry.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.crdlcacheadminfrontend.codeLists.models
+
+import play.api.libs.json.*
+
+case class CodeListEntry(
+  key: String,
+  value: String,
+  properties: Map[String, String]
```

Comment:
I think this can be more than a `Map[String, String]` - e.g. some of the codelists contain `Boolean` properties - that's why we use `JsObject` in crdl-cache for the `properties`

Reply:
I guess the concern that I have is that the JSON deserialization process may fail for entries that have a Boolean property, but perhaps the deserialization is less picky than I thought